### PR TITLE
Add Grafana troubleshooting dashboards

### PR DESCRIPTION
**Pull request by RiskyAdventure**
Issue #, if available:

Description of changes:
  First version of low level troubleshooting dashboards for the control plane.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: substrate/monitoring/Shancor-KCM-v0.5.json [hunk 1]
```diff
@@ -0,0 +1,910 @@
+{
```
Comment (prateekgogia) [inline substrate/monitoring/Shancor-KCM-v0.5.json:1]: Getting the following error when importing KCM dashboard from Grafana console-
```
JSON -> JS Serialization failed: JSON.parse: unexpected character at line 7 column 1 of the JSON data
```

File: substrate/monitoring/Shancor-Node-CPU-Troubleshooter-v0.5.json [hunk 2]
```diff
@@ -0,0 +1,760 @@
+{
```
Comment (prateekgogia) [inline substrate/monitoring/Shancor-Node-CPU-Troubleshooter-v0.5.json:1]: Throttled CPU Seconds (Bad Limits) panel has an error-
```
_.viz is undefined
```

**Review by prateekgogia — COMMENTED** [review]
Hey thanks for contributing this.

Some high level feedback-
- Not able to add the dashboards by creating them as config maps need to rely on Grafana console to import these dashboards.
- If we can rename these files (remove username Shancor) and tags making it more generic will be helpful

**Comment by prateekgogia** [comment]
Getting error in kubelet-troubleshooting dashboard 
```
JSON -> JS Serialization failed: Unexpected token < in JSON at position 6
```
Can we skip adding spaces to the file names, as we need to make sure the format is correct when using it locally.

**Comment by RiskyAdventure** [comment]
All updates pushed as requested.

**Review by prateekgogia — APPROVED** [review]
Tested using the UI and importing the dashboard works. LGTM

**Pull request merged**